src/commands/project/watch.ts

```diff
@@ -90,7 +90,7 @@ function watch(project: string, cmdFlags: Flags, creds: Credentials|undefined, o
   }
   const watch = () => {
     // logger.log("Opening new watcher")
-    watcher = chokidar.watch(project, { ignoreInitial: true, followSymlinks: false, usePolling: false, useFsEvents: false })
+    watcher = chokidar.watch(project, { ignoreInitial: true, followSymlinks: false, usePolling: false, useFsEvents: false, ignored: getExclusionList() })
```

Comment:
Nice touch to use chokidar's own `ignored` flag here, but, are you sure of the semantics (I haven't used it myself).

Reply:
I tested it, we'll have unit tests soon!